release-notes/swan-lake-alpha4-release-note.md

```diff
@@ -56,8 +56,69 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### New Features
 
+##### Support for Relational Expressions With All Ordered Types
+
+Relational expressions (`<`, `>`, `<=`, and `>=`) are supported with all [ordered types](https://ballerina.io/spec/lang/draft/v2020-12-17/#ordering). The static type of both operands must belong to the same ordered type.
+
+##### Support for Inferring the Argument of a Dependently-Typed Function from the Contextually-Expected Type
+
+When the default value of a `typedesc` parameter of a dependently-typed function is `<>` and an argument is not provided for the parameter when calling the function, the argument will be inferred from the contextually-expected type of the function call.
+```ballerina
+function func(typedesc<anydata> td = <>) returns td = external;
+
+public function main() {
+    // The argument for `td` is inferred to be `int`.
+    int value = func();
+}
+```
+
 #### Improvements
 
+##### Dependently-Typed Lang Library Functions that Infer the Argument from the Contextually-Expected Type
```

Comment:
```suggestion
##### 
Improvements to the Dependently-typed Lang Library Function that Infer the Argument from the Contextually-expected type. 
```

Reply:
The improvement was to infer the argument. How about something like

```md
Improvements to Dependently-Typed Lang Library Functions to Infer the Argument from the Contextually-Expected Type
```

Reply:
+1